testing/integration/proxy_url_test.go

```diff
@@ -382,42 +709,48 @@ func TestProxyURL(t *testing.T) {
 			err = fixture.EnsurePrepared(ctx)
 			require.NoError(t, err, "SetupTest: fixture.Prepare failed")
 
-			installProxyURL := ""
-			if tt.args.enrollProxyName != "" {
-				require.Containsf(t, proxies, tt.args.enrollProxyName, "Proxy %q to be used for enrolling is missing from the map of proxies", tt.args.enrollProxyName)
-				installProxyURL = proxies[tt.args.enrollProxyName].LocalhostURL
-			}
-
-			require.NotNil(t, tt.enrollmentURL, "testcase must define an enrollmentURL getter")
-			enrollmentURL := tt.enrollmentURL(fleet)
-
 			out, err := fixture.Install(
 				ctx,
 				&integrationtest.InstallOpts{
-					Force:          true,
-					NonInteractive: true,
-					Insecure:       true,
-					ProxyURL:       installProxyURL,
+					Force:                  true,
+					NonInteractive:         true,
+					Insecure:               args.insecure,
+					ProxyURL:               args.proxyURL,
+					CertificateAuthorities: args.certificateAuthorities,
+					Certificate:            args.certificate,
+					Key:                    args.key,
```

Comment:
[not a blocker]
Those should be on `integrationtest.EnrollOpts`

Reply:
Done in bc6c4bb8fdec89396ba0bd1dfe21fdcae7163ce8